examples/specs/stacked_bar_opacity2.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
rename this file to `stacked_bar_opacity` after correct the filename for `stacked_bar_opacity`.